dynamodb/toddl.go

```diff
@@ -28,6 +28,19 @@ import (
 // Spanner. It uses the source schema in conv.SrcSchema, and writes
 // the Spanner schema to conv.SpSchema.
 func schemaToDDL(conv *internal.Conv) error {
+	// Tracks Spanner names that have been used for indexes. We use this to ensure we generate unique names when
+	// we map from DynamoDB to Spanner since we want Spanner table names and index names to be distinct.
+	usedNames := make(map[string]bool)
+	// We need to pre-populate usedNames with Spanner table
+	// names to handle collision with table names, foreign key names and index names.
```

Comment:
nit; Formatting looks a little weird in the sense that newline started all of a sudden after Spanner table. Can you fix it?